controllers/topolvm_node.go

```diff
@@ -177,6 +177,8 @@ func getNodeDaemonSet(lvmCluster *lvmv1alpha1.LVMCluster, namespace string, init
 	}
 	labels := map[string]string{
 		"app": topolvmNodeName,
```

Comment:
Are we using this existing label anywhere? If not I would like to remove it.

Reply:
can be removed